### PR TITLE
@l2success / @kanaabe => Article styles updates

### PR DESCRIPTION
**Pull request by eessex**
Addresses [GROW-719](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=GROW&modal=detail&selectedIssue=GROW-719)

Sorry for the insane number of files -- mostly snapshots

- Updates Palette
- Standardizes bullets in new size (8px)
- Changes "More In" to "More in"
- Refactors `AuthorDate` into separate components, `Author` and `Date`
- Updates font sizes in `RelatedArticleCanvas`
- Uses Palette Sans font for:
     - Author
     - Date
     - FeatureHeader deck
     - Vertical 

Also addresses some consistency issues with byline and vertical, so all layouts are using the same components

### CODE REVIEW

File: src/Components/Publishing/Byline/Author.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import React from "react"
+import styled from "styled-components"
+import { getAuthorByline } from "../Constants"
+import { BylineLayout } from "../Typings"
+
+interface AuthorProps {
+  authors?: any
+  color?: string
+  layout?: BylineLayout
+  size?:
```
Comment (l2succes) [inline src/Components/Publishing/Byline/Author.tsx]: @eessex Doesn't that size type live in palette already? Why not pull it from there?

Comment (eessex) [reply src/Components/Publishing/Byline/Author.tsx]: Oh meant to leave a todo here- is not exported, going to change in palette and replace here and other instances

File: src/Components/Publishing/Byline/Share.tsx [hunk 2]
```diff
@@ -71,29 +71,49 @@ export class Share extends React.Component<Props, null> {
...
-          <IconSocialEmail color={color} />
-        </IconWrapper>
+        {hasLabel && (
+          <ShareLabel size="3t" weight="medium">
+            Share
+          </ShareLabel>
+        )}
+        {this.getIcon("facebook")}
+        {this.getIcon("twitter")}
+        {this.getIcon("email")}
```
Comment (kanaabe) [inline src/Components/Publishing/Byline/Share.tsx]: Really liking these little refactors! 👍

File: src/Components/Publishing/Header/ClassicHeader.tsx [hunk 3]
```diff
@@ -37,30 +37,26 @@ const ClassicHeaderContainer = styled.div`
...
     width: 100%;
     margin: 0 auto;
     padding-bottom: 30px;
+    ${garamond("s19")};
   }
 
-  ${pMedia.xl`
-    padding: 0 20px;
-  `} ${pMedia.xs`
+  ${pMedia.xl`padding: 0 20px;`} ${pMedia.xs`
```
Comment (kanaabe) [inline src/Components/Publishing/Header/ClassicHeader.tsx]: Maybe give these a little breathing space with `;`

File: src/Components/Publishing/Sections/FullscreenViewer/__tests__/FullscreenViewer.test.tsx [hunk 4]
```diff
@@ -22,6 +22,6 @@ it("closes the viewer on ESC keydown", () => {
   const viewer = mount(
     <FullscreenViewer images={Images} show onClose={onClose} />
   )
-  viewer.simulate("keyDown", { keyCode: 27 })
-  expect(onClose).toBeCalled()
+  // viewer.simulate("keyDown", { keyCode: 27 })
+  // expect(onClose).toBeCalled()
```
Comment (kanaabe) [inline src/Components/Publishing/Sections/FullscreenViewer/__tests__/FullscreenViewer.test.tsx]: Intentional?

Comment (eessex) [reply src/Components/Publishing/Sections/FullscreenViewer/__tests__/FullscreenViewer.test.tsx]: Good catch, thanks!

File: src/Components/Publishing/Byline/Author.tsx (hunk 1 above)
Comment (eessex) [reply src/Components/Publishing/Byline/Author.tsx]: This has been updated to use Palette types

**Comment by kanaabe** [comment]
LGTM, feel free to merge yourself when ready. This was 💯

**Pull request merged**